MonkeyWrench.Database/Extensions/DBLogin_Extensions.cs

```diff
@@ -69,33 +71,41 @@ public static DBLogin Login (DB db, string login, string password, string ip4, b
 			return result;
 		}
 
-		public static void LoginOpenId (DB db, LoginResponse response, string email, string ip4)
+		public static void Login (DB db, LoginResponse response, string email, string ip4, List<string> userOrgs, bool useGitHub = false, string gitHubLogin = "")
 		{
-			if (string.IsNullOrEmpty (Configuration.OpenIdProvider) && string.IsNullOrEmpty (Configuration.OauthClientId))
-				throw new Exception ("No OpenId provider available");
+			string [] specs;
 
-			if (string.IsNullOrEmpty (Configuration.OpenIdRoles))
-				throw new Exception ("No OpenId roles specified");
+			string username = useGitHub ? gitHubLogin : email;
```

Comment:
Actually I thought about this some more and talked with @duncanmak about it to make sure this filter made sense. I don't think it does, because `username` is not used as a way to authenticate. It's only used as a way to give the user a username. The GitHub check is based on their organization, so they have to be in a organization that is listed in the Wrench config file. Email is used if you are using Google/OpenID. And you can only use email if the `useGitHub` flag is set to false.

I don't think there is an attach issue that would happen here. And while the filter is harmless, it can be confusing because there is no need for it and can make people looking at the code think that `username` means something it's not.

I removed the filter here [https://github.com/mono/monkeywrench/pull/74/commits/cd6acb7ede2f142d723ed0f28c6ae4ee1e708e8c] .